bundles/org.openhab.binding.hue/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -62,4 +62,44 @@
 		</config-description>
 	</bridge-type>
 
+	<!-- CLIP 2 Bridge -->
+	<bridge-type id="clip2">
+		<label>Hue API v2 Bridge</label>
+		<description>The Hue Bridge represents a Philips Hue Bridge supporting API v2.</description>
+
+		<channels>
+			<channel id="scene" typeId="scene"/>
+		</channels>
+
+		<properties>
+			<property name="vendor">Signify</property>
+		</properties>
+		<representation-property>serialNumber</representation-property>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>network-address</context>
+				<label>Network Address</label>
+				<description>Network address of the Hue Bridge.</description>
+			</parameter>
+			<parameter name="applicationKey" type="text">
+				<context>password</context>
+				<label>Application Key</label>
+				<description>A registered Hue Bridge application key that allows access to the API.</description>
+			</parameter>
+			<parameter name="checkSeconds" type="integer" min="600" step="60" unit="s">
```

Comment:
Since minimum is 600 (10 minutes) and step is "60" (1 minute), the actual granularity is **minutes**. Therefore, the parameter could be simplified if renamed to **checkMinutes** and representing minutes.

Reply:
Understood. I was trying to maintain consistency with v1. But probably better to change.